src/site/stages/build/page-builder/schemas/#page.js#

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable camelcase */
+
+module.exports = () => ({
+  type: 'object',
+  properties: {
+    title: { type: 'string' },
+    field_intro_text: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          value: { type: 'string' },
+        },
+      },
+    },
+    field_description: {
+      type: 'array',
+      items: {
+        type: 'object',
+        properties: {
+          value: { type: 'string' },
+        },
+      },
+    },
```

Comment:
Is there a reason why this isn't a `{ $ref: 'generic-nested-string' }`?

Reply:
Blast. Because it was a temp file that shouldn't have gotten committed. :shakes-fist:

Reply:
I didn't even notice the filename!